cli/src/pcluster/config/cluster_config.py

```diff
@@ -277,6 +353,160 @@ def __init__(
         # TODO register validators
 
 
+class SharedStorageConfig(Config):
+    """Represent the Shared Storage configuration."""
+
+    def __init__(
+        self,
+        mount_dir: str,
+        ebs_settings: EbsConfig = None,
+        efs_settings: EfsConfig = None,
+        fsx_settings: FsxConfig = None,
+    ):
+        super().__init__()
+        self.mount_dir = mount_dir
+        if ebs_settings:
+            self.ebs_settings = ebs_settings
+        elif efs_settings:
```

Comment:
Does the cluster allow multiple file systems? Since I see the test template `slurm.full.yaml` contains 3 file systems, here just enables to assign one to cluster

Reply:
I see this is defined in cluster schema as many, never mind